src/User32.Desktop/User32+DESKTOP_ACCESS_MASK.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Diagnostics;
+
+    /// <content>
+    /// Contains the <see cref="DESKTOP_ACCESS_MASK"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// The <see cref="Kernel32.ACCESS_MASK"/> type is a bitmask that specifies a set of access rights in the access mask of an access control entry.
+        /// /// This is a set of Specific Rights bitmask that defines access rights and security for desktop objects
+        /// </summary>
+        /// <remarks>
+        /// <see cref="Kernel32.ACCESS_MASK"/> is quite well described here: http://blogs.msdn.com/b/openspecification/archive/2010/04/01/about-the-access-mask-structure.aspx
+        /// Desktop security and access rights are quite well described here: https://msdn.microsoft.com/en-us/library/windows/desktop/ms687391(v=vs.85).aspx
+        /// </remarks>
+        public struct DESKTOP_ACCESS_MASK : IComparable, IFormattable, IComparable<DESKTOP_ACCESS_MASK>, IEquatable<DESKTOP_ACCESS_MASK>, IComparable<Kernel32.ACCESS_MASK>, IEquatable<Kernel32.ACCESS_MASK>
```

Comment:
Well if we keep with SemVer, any incompatible change to API mean that we must increment the major version :wink: as for maintainance except if microsoft decide on releasing a PInvoke lib of their own (And keep it up longer that the defunct "Windows API Code Pack") I don't see the project stopping.

*And no structs sadly don't have any inheritance feature, they can implement interfaces but they are boxed each time to use them (Can have perf consequences depending on usage)*

I don't know if there is a perfect solution here, the classical solution is to use classes instead of enums (Like old school java) but we need a solution that works with the marshaler (Or it could involve one more code transformation like FriendlyAttribute).